pkg/jsonclient/client.go

```diff
@@ -58,18 +55,12 @@ func MakeRequest(ctx context.Context, client *http.Client, url string, headers h
 	}
 	defer r.Body.Close()
 
-	logger.Debugf("http status: %s (%d)", http.StatusText(r.StatusCode), r.StatusCode)
-	for k, v := range r.Header {
-		logger.Debugf("response header: %q: %v", k, v)
```

Comment:
this impacts our CLIs for things like get-token, and we'll not get any throttling info dumped any longer...